packages/lib/modules/pool/actions/PoolActionsLayout.tsx

```diff
@@ -1,49 +1,23 @@
 'use client'
 
-import { Box, VStack, Card, useColorModeValue } from '@chakra-ui/react'
 import { PropsWithChildren } from 'react'
-import { PoolActionsNav } from './PoolActionsNav'
 import { usePool } from '../PoolProvider'
-import { usePoolRedirect } from '../pool.hooks'
 import { PoolName } from '../PoolName'
-import { NavBar } from '@repo/lib/shared/components/navs/NavBar'
+import { getPoolPath } from '@repo/lib/modules/pool/pool.utils'
+import { ModalActionsLayout } from '@repo/lib/shared/components/layout/ModalActionsLayout'
 
 type Props = PropsWithChildren
 
 export function PoolActionsLayout({ children }: Props) {
   const { pool } = usePool()
-  const { redirectToPoolPage } = usePoolRedirect(pool)
-
-  const bg = useColorModeValue('blackAlpha.800', 'blackAlpha.700')
-  const blur = useColorModeValue('blur(8px)', 'blur(5px)')
 
   return (
-    <Box
-      bg="transparent"
-      left={0}
-      onClick={redirectToPoolPage}
-      pos="absolute"
-      top={0}
-      w="full"
-      zIndex={100}
+    <ModalActionsLayout
+      chain={pool.chain}
```

Comment:
1. `ModalActionsLayout` isn't great naming because it's not really a modal.
   Please rename `FocussedActionLayout` and `FocussedActionNav`.
2. Because you've nested all the logic in `ModalActionsLayout` please get rid of
   `PoolActionsLayout`. This wrapper isn't needed.